docs/src/views/ColorPalette.tsx

```diff
@@ -0,0 +1,169 @@
+import { Divider, Grid, Header, Icon, Text } from '@stardust-ui/react'
+import * as _ from 'lodash'
+import * as React from 'react'
+import * as color from 'tinycolor2'
+
+import Slot from 'src/components/Slot/Slot'
+import {
+  baseColors,
+  ColorPalette,
+  naturalPalettes,
+  statePalettes,
+  StatePalettes,
```

Comment:
Should palettes be plural - or like ColorPalette

Reply:
Sure. However, I don't think that will call it `Palette` at all, too many palettes there. Naming problem :)